### DISCUSSION TITLE
您好，开放测试的200道题，有类似于txt或者json格式的数据吗

### DISCUSSION THREAD

**Discussion by eyuansu62** [ymcui/Chinese-LLaMA-Alpaca#845] [General]
您好，开放测试的200道题，有类似于txt或者json格式的数据吗

**Comment by airaria** [comment]
你可以从[https://github.com/ymcui/Chinese-LLaMA-Alpaca/tree/main/examples/q8_7b-13b-p7b](https://github.com/ymcui/Chinese-LLaMA-Alpaca/tree/main/examples/q8_7b-13b-p7b)里的markdown文件中中自行抽取问题相关字段。
